server/src/controllers/User/resolver.ts

```diff
@@ -39,6 +40,37 @@ export class UserWithPermissionsResolver {
   async userInformation(
     @Ctx() ctx: ResolverCtx,
   ): Promise<UserInformation | null> {
```

Comment:
`UserInformation` is a bit vague, particularly now we also have `UserData`.  I think `UserForDownload` and `UserProfile`, respectively, would be better, so that it's obvious what they're used for.